src/applications/pact-act/tests/utilities/customize-title.unit.spec.js

```diff
@@ -4,11 +4,13 @@ import { customizeTitle } from '../../utilities/customize-title';
 describe('utilities: customize title', () => {
   it('should return the correct title with the H1', () => {
     expect(customizeTitle('This is a test H1')).to.equal(
-      'This is a test H1 | PACT Act | Veterans Affairs',
+      'Learn how the PACT Act may effect you | Veterans Affairs',
     );
   });
 
   it('should return the correct title without an H1', () => {
-    expect(customizeTitle()).to.equal('PACT Act | Veterans Affairs');
+    expect(customizeTitle()).to.equal(
+      'Learn how the PACT Act may effect you | Veterans Affairs',
```

Comment:
@laflannery Should this be "affect"?

Reply:
I believe you're right. I copy pastad from the ticket. Let's verify with @laflannery

Reply:
@randimays is absolutely correct. I **should** have copy pasted from the page lol

Reply:
Fixed, added screenshot with updated text